.github/workflows/build.yml

```diff
@@ -74,7 +74,7 @@ jobs:
                 run: bin/phpspec run --format=dot
 
             -   name: Run tests (phpunit)
-                run: ./vendor/bin/phpunit -v
+                run: ./vendor/bin/phpunit
```

Comment:
Verbose option has been removed in version 10 (https://github.com/sebastianbergmann/phpunit/blob/10.0.0/ChangeLog-10.0.md) 

Is this ok to just remove it?